app/javascript/controllers/organization_form_controller.js

```
@@ -0,0 +1,227 @@
+    bindCounter(inputId, counterId, maxLen) {
+        const el = document.getElementById(inputId);
+        const counter = document.getElementById(counterId);
+        if (!el || !counter) return;
+        const update = () => {
+            const remaining = maxLen - el.value.length;
+            counter.textContent = `${remaining} characters remaining`;
+            counter.classList.toggle('text-danger', remaining <= Math.floor(maxLen * 0.15));
+        };
+        update();
+        el.addEventListener('input', update);
+    }
```

ESLint

[error] Expected 'this' to be used by class method 'bindCounter'.

(class-methods-use-this)